tests/unit/core/server/test_DHTHashAnnouncer.py

```diff
@@ -0,0 +1,50 @@
+from lbrynet.core.server.DHTHashAnnouncer import DHTHashAnnouncer,DHTHashSupplier
+from lbrynet.core.utils import random_string
+from lbrynet.core import log_support
+from twisted.trial import unittest
+from twisted.internet import defer,task
+import logging
+import sys
+import os,binascii
```

Comment:
import should be ordered by

```
system

third party

local
```

and alphabetical within each grouping.  Don't have multiple imports on the same line.  Instead do
```
import binascii
import os
```